internal/resources/akscluster/schema.go

```diff
@@ -205,6 +205,13 @@ var ClusterConfig = &schema.Resource{
 			MaxItems:    1,
 			Elem:        AutoUpgradeConfig,
 		},
+		identityConfigKey: {
```

Comment:
Thanks for recording all the test results with detail :-) I was just wondering, since this is a new field in the schema does it not involve any changes to request/response mapping in the resource/data-source?

Reply:
@ramya-bangera 
Can you please clarify what you mean by 'resource/data-source' as that does not seem to map to a file in the repo?

It could be added as an example output to 'examples/data-sources/akscluster/cluster.tf' to go alongside the 'location' example.

```
// Read the managed identity of the cluster received as a data source
output "managed_identity" {
  value = data.tanzu-mission-control_akscluster.tf_aks_cluster.spec[0].config[0].identity_config
}
```

It could be added to 'resource_templates/akscluster.tf' if needed, but would not be a required field.

It is not needed in 'examples/resources/akscluster/minimal_cluster.tf' as it is not required to be set by the user as part of the minimum fields needing to be provided as it defaults to system when not specified.

Reply:
I was asking for the request mapping of this field which I can see in file internal/resources/akscluster/akscluster_mapper.go at L143, I missed to see this in the first review.